packages/qdrant-loader/tests/integration/core/state/conftest.py

```
@@ -0,0 +1,37 @@
+        return
+
+    try:
+        from testcontainers.postgres import PostgresContainer
+    except ImportError:
+        pytest.skip("testcontainers not installed; set TEST_POSTGRES_URL to run PG tests")
+        return
+
+    try:
+        container = PostgresContainer("postgres:16-alpine", driver="asyncpg")
+        container.start()
+    except Exception as exc:  # Docker not running / image pull failed
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)